### ISSUE TITLE
session 3 - review exercise to connect storage to Linux VM #14

### ISSUE THREAD

**Issue by billspat** [MSU-ICER/cloudcomputingfellowship#15]
review Azure tutorial to connect storage to Linux using NFS (and or SMB).    Make recommendation to create version of this specifically for the cloud fellowship if it's hard to follow but valuable. 

also make it not optional and an alternative to the Windows + storage connection tutorial

**Comment by emtelgen** [comment]
notes:
first linux exercise is workable (i must have been doing something wrong earlier this week, not sure what, but i went through it again and it works)
I ran into an issue with the folder being "public" to ssh with the .pem file. When that happened i just moved the .pem file to my downloads folder and it worked. People might run into this issue

second linked exercise:  These are fixes to the issues I was coming into
USE LINUX VM (i was using my WSL and it was not working, it works in the Linux VM)
can not use the fileshare you created in the first linux exercise (that is nfs and the tutorial is for smb) - need to make a new fileshare and set it as smb
I was able to complete tutorial up to "Dynamically mount with autofs" this was not working for me
